CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [UNRELEASED] NEPTUNE 1.0.1
```

Comment:
`1.1.0`

Reply:
No, it's hidden feature, not documented for enduser.

Reply:
So let's move it to Changes instead or even do not expose it via Changelog :wink: